### ISSUE TITLE
bug: langfuse js sdk `langfuse-core@3.30.1` cause build error info

### ISSUE THREAD

**Issue by arvinxx** [langfuse/langfuse#4346]
### Describe the bug

langfuse js sdk cause build time error with 3.30.1

```
✓ (serwist) Bundling the service worker script with the URL '/sw.js' and the scope '/'...
 ⚠ Compiled with warnings
./node_modules/langfuse-core/lib/index.mjs
Module not found: Can't resolve 'fs' in '/vercel/path0/node_modules/langfuse-core/lib'
Import trace for requested module:
./node_modules/langfuse-core/lib/index.mjs
./node_modules/langfuse/lib/index.mjs
./src/libs/traces/index.ts
./src/app/(backend)/webapi/trace/route.ts
./node_modules/next/dist/build/webpack/loaders/next-app-loader.js?name=app%2F(backend)%2Fwebapi%2Ftrace%2Froute&page=%2F(backend)%2Fwebapi%2Ftrace%2Froute&pagePath=private-next-app-dir%2F(backend)%2Fwebapi%2Ftrace%2Froute.ts&appDir=%2Fvercel%2Fpath0%2Fsrc%2Fapp&appPaths=%2F(backend)%2Fwebapi%2Ftrace%2Froute&pageExtensions=tsx&pageExtensions=ts&pageExtensions=jsx&pageExtensions=js&basePath=&assetPrefix=&nextConfigOutput=&preferredRegion=&middlewareConfig=e30%3D!./src/app/(backend)/webapi/trace/route.ts?__next_edge_ssr_entry__
```

### To reproduce

just one click deploy our open source project lobe-chat to vercel, you can find the error message in the build log.

https://github.com/lobehub/lobe-chat?tab=readme-ov-file#a-deploying-with-vercel-zeabur--sealos-or-alibaba-cloud

### Are you interested to contribute a fix for this bug?

No

**Comment by hassiebp** [comment]
Hi @arvinxx - thanks a lot for reporting this. Could you please try again with version `3.30.2`? Are you using vercel edge runtime?

**Comment by arvinxx** [comment]
@hassiebp hi, don't work with 3.30.3

**Comment by arvinxx** [comment]
> Are you using vercel edge runtime?

yes

**Comment by hassiebp** [comment]
@arvinxx - thanks for your feedback, we have added a fix for edge runtimes in [v3.30.4](https://github.com/langfuse/langfuse-js/releases/tag/v3.30.4), please upgrade and kindly let us know whether all works ok! 🙏🏾
